celerdata/assets/logs/celerdata.yaml

```diff
@@ -0,0 +1,47 @@
+id: celerdata
+metric_id: celerdata
+backend_only: false
+facets: null
+pipeline:
+  type: pipeline
+  name: CelerData
+  enabled: true
+  filter:
+    query: source:celerdata
+  processors:
+    - type: grok-parser
+      name: Parsing Log
+      enabled: true
+      source: message
+      samples:
+        - 2024-03-28 10:21:53,960 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():282] create built-in role cluster_admin[-3]
+        - 2024-03-28 18:58:43,380 INFO (nioEventLoopGroup-4-3|154) [RestBaseAction.handleRequest():73] receive http request. url=/api/health
+        - 2024-04-10 10:32:23.937+08:00 INFO (stateChangeExecutor|95) [StateChangeExecutor.runOneCycle():179] finished to transfer FE type from INIT to LEADER
+        - I0328 10:22:09.303526 1066 olap_server.cpp:844] try to perform path gc by tablet!
+      grok:
+        supportRules: ""
+        matchRules: >-
+          fe.log %{date("yyyy-MM-dd HH:mm:ss,SSS"):timestamp} %{word:loglevel}
```

Comment:
%{data}\|%{number}\
You could use the operator `notSpace` instead of `data` here, as I don't think any space would be tolerated here?

Reply:
done